### DISCUSSION TITLE
modification/correction/improvement of the description of an exercise

### DISCUSSION THREAD

**Discussion by clafalco** [wger-project/flutter#338] [Q&A]
HI! Is it possible to correct and add descriptions to an already inserted exercise?

**Comment by rolandgeider** [comment]
Hi! Yeah that's not only possible but also encouraged. At the moment you need to have an account at least 14 days old and confirmed your email (I think this is only shown explicitly when you try to submit a new one)

**Reply by clafalco** [reply]
HI!! thank you for your answer, my email is verified, and my account has been active for over 20 days, what I can do is add exercises and translate into my language.
I ask you, if in addition to this it is possible to modify and perfect correct those already written.
thanks

**Reply by rolandgeider** [reply]
yeah, that should be possible as well. Do you only see the name/description fields when editing?

**Reply by clafalco** [reply]
yes here is what i see, i can only translate, but not edit the original


![image](https://user-images.githubusercontent.com/122402237/216781414-d0173777-785c-4e95-b6e0-f9a89b526a72.png)

**Reply by rolandgeider** [reply]
ahh, I thought you wanted to change things like the muscles, equipment and so on. You should be able to edit the original by changing the exercise language to English

**Reply by clafalco** [reply]
ok thanks I understand, but as you said is it possible to change muscles and equipment too?

**Reply by rolandgeider** [reply]
it should be possible, there should be some fields at the bottom on that page similar to the ones when submitting a new one. I'll check tomorrow if there's some bug that always hides them or something like that

**Reply by clafalco** [reply]
thank you very much :+1: , I don't see any other fields below (only the ability to add images)

**Reply by rolandgeider** [reply]
Then there's something missing. I've made an issue so this doesn't get lost: https://github.com/wger-project/react/issues/614
